api/api-data-sharing-settings-v1/src/main/java/com/thoughtworks/go/apiv1/datasharing/settings/DataSharingSettingsControllerV1Delegate.java

```diff
@@ -98,6 +103,11 @@ public String patchDataSharingSettings(Request request, Response response) throw
         return handleCreateOrUpdateResponse(request, response, dataSharingSettingsService.get(), result);
     }
 
+    public String getDataSharingNotificationForCurrentUser(Request request, Response response) {
+        boolean shouldAllow = dataSharingNotification.allowNotificationFor(currentUsername());
+        return jsonizeAsTopLevelObject(request, writer -> writer.add("allow", shouldAllow));
```

Comment:
can we say `show_notification` instead of `allow`?

Reply:
ok